pkg/codegen/internal/test/testdata/types.json

```diff
@@ -0,0 +1,224 @@
+{
+  "name": "typetests",
+  "version": "0.0.1",
+  "meta": {
+    "moduleFormat": "(.*)"
+  },
+  "config": {},
+  "types": {
+    "typetests::primitives": {
```

Comment:
I can understand what this file asserts, so that is good. Does it cover all primitives? Where do we define all the primitive types, could we link it here?

Reply:
It covers all of the simple primitives. I'll link them in the description for this case.

Reply:
Awesome thanks.